packages/service-clients/azure-client/src/AzureClient.ts

```diff
@@ -57,6 +57,10 @@ const MAX_VERSION_COUNT = 5;
 const azureClientFeatureGates = {
 	// Azure client requires a write connection by default
 	"Fluid.Container.ForceWriteConnection": true,
+
+	// These settings will ensure the GC op isn't sent, since it can't be processed by V1 clients
+	"Fluid.GarbageCollection.RunSweep": false,
```

Comment:
Should we also disable tombstone? Otherwise, there may be tombstone errors and / or failures? At least tombstone enforcement should be disabled.

Reply:
I think that's fine. We don't expect that to happen - we're not shipping any known bugs and there's no request pattern. 

This is about preventing the GC op from causing v1 clients to crash.

Reply:
Okay. I was thinking from the perspective of being as safe as possible since we don't really care about GC for these clients while they are upgrading right.

Reply:
Gotcha. Let's discuss offline

Reply:
Decided to disable Tombstone Enforcement as well for a simpler story